tools/site_cobble/rdl_pkg/models.py

```diff
@@ -68,15 +70,23 @@ def __init__(self, **kwargs):
     def packed_fields(self):
         return [x for x in self.fields if not isinstance(x, ReservedField)]
 
+    @property
+    def has_reset_definition(self):
+        # Get the reset value for all the fields. If we don't see None in any of them we have defined reset behavior
+        a = [x.get_property('reset') for x in self.fields if not isinstance(x, ReservedField)]
+        return False if None in a else True
+
     def elaborate(self):
         """
         Register elaboration consists of sorting the defined fields by the
         low index of the field. We then loop through the fields and
         determine the largest contiguous gaps in the definitions and creating
         ReservedFields that fill into these spaces. These are accumulated in
-        a gaps variable, and the gaps and fields are contactenated and
+        a gaps variable, and the gaps and fields are concatenated and
         re-sorted by low index again at the end.
         """
+        if self.width != 8:
+            raise UnsupportedRegisterSizeError(f"We only support 8bit registers at this time. Register {self.name} has a width of {self.width}")
```

Comment:
Thanks for adding this. For others, a bit more context on https://github.com/oxidecomputer/cobalt/issues/27